SalesforceCommerceCloud/pwa-kit
# [Retail App] Shipping method description overflows the price section *FIXED*

# Description
Changed the Shipping options description text width as 70% for base, 65% for md and 60% for lg devices to fix the overflow problem

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added the width property in the Text tag:- width={{base: '70%', md: '65%', lg: '60%'}}

# How to Test-Drive This PR

1. Add a product to the cart
2. Check out
3. Add delivery address
4. In shipping options, try resizing the screen to check for individual devices

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization



## Screenshot
<img width="1792" alt="Screenshot 2021-12-10 at 1 22 13 AM" src="https://user-images.githubusercontent.com/83722798/145467489-5421f742-8c5d-4462-a011-6a4335d2078d.png">
